server/handlers/default.go

```diff
@@ -199,7 +199,7 @@ func getSnapshot(ctx context.Context, w http.ResponseWriter, logger ctxu.Logger,
 	out, err := snapshot.GetOrCreateSnapshot(gun, store, cryptoService)
 	if err != nil {
 		switch err.(type) {
-		case *storage.ErrNoKey, *storage.ErrNotFound:
+		case *storage.ErrNoKey, storage.ErrNotFound:
```

Comment:
I'm slowly fixing all our errors as I encounter them. I just crossed paths with ErrNotFound writing some tests so I fixed it. Without very good reasons, none of our errors should be pointers, but we were all still learning Go when we wrote some of this.